mailer/tests.py

```diff
@@ -593,6 +593,35 @@ def test_message_log(self):
             self.assertEqual(log.to_addresses, [])
             self.assertEqual(log.subject, "")
 
+    def test_message_str(self):
+        with self.settings(MAILER_EMAIL_BACKEND="django.core.mail.backends.locmem.EmailBackend"):
+            mailer.send_mail("Subject Msg", "Body", "msg1@example.com", ["rec1@example.com"])
+
+            self.assertEqual(Message.objects.count(), 1)
+
+            msg = Message.objects.all()[0]
```

Comment:
if you've already checked that there is exactly one message, this could be written as `msg = Message.objects.get()`